src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -82,10 +83,31 @@ export const AppRouter: React.FC = () => {
           {/* @debt The /login route doesn't work since we added non-defaulted props to the Login component */}
           {/*<Route path="/login" component={Login} />*/}
 
-          <Route path="/v/:venueId" component={VenueLandingPage} />
+          <Route
+            path="/v/:venueId"
+            component={() => (
+              <Provided withWorldUsers withRelatedVenues>
+                <VenueLandingPage />
+              </Provided>
+            )}
+          />
           <Route path="/e/:step/:venueId" component={VenueEntrancePage} />
-          <Route path="/in/:venueId/admin" component={VenueAdminPage} />
-          <Route path="/in/:venueId" component={VenuePage} />
+          <Route
+            path="/in/:venueId/admin"
+            component={() => (
+              <Provided withRelatedVenues>
+                <VenueAdminPage />
+              </Provided>
+            )}
+          />
+          <Route
+            path="/in/:venueId"
+            component={() => (
+              <Provided withWorldUsers withRelatedVenues>
+                <VenuePage />
+              </Provided>
+            )}
+          />
```

Comment:
This `component={() => (` syntax is invalid and will cause constant re-renders.

The 'nested' method I used in my described solution is the canonically best/correct way to do this.

**Further Reading:**

- https://reactrouter.com/web/api/Route/route-render-methods
  - > **The recommended method of rendering something with a `<Route>` is to use children elements**, as shown above. There are, however, a few **other methods** you can use to render something with a `<Route>`. These are provided mostly for **supporting apps that were built with earlier versions of the router before hooks were introduced.**
  - https://reactrouter.com/web/api/Route/component
    - > When you use component (instead of render or children, below) the router uses `React.createElement` to create a new React element from the given component. That means **if you provide an inline function to the component prop, you would create a new component every render. This results in the existing component unmounting and the new component mounting instead of just updating the existing component.** When using an inline function for inline rendering, use the `render` or the `children` prop (below).
- https://reactrouter.com/web/api/Route/render-func
  - > This allows for convenient inline rendering and wrapping **without the undesired remounting explained above.**

Reply:
Went with the first working solution, same as https://github.com/sparkletown/sparkle/pull/1769/files#r667278276

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1769/commits/c58428242104ab1c0becf206460260f3c346a754